src/main/scala/strawman/collection/mutable/Seq.scala

```diff
@@ -0,0 +1,76 @@
+package strawman
+package collection
+package mutable
+
+trait Seq[A] extends strawman.collection.Seq[A] with Growable[A] {
```

Comment:
Is there something wrong with the "obvious" solution of having the `Growable` trait implemented on only a subset of collections?  So we'd have `GrowableSeq` that extends both `Growable` and `Seq`, or alternatively `UpdatingSeq` which only allows update, and have `Seq` be `UpdatingSeq with Growable`?